### ISSUE TITLE
OTA Update

### ISSUE THREAD

**Issue by D34TH5TR1K3** [mrfaptastic/ESP32-HUB75-MatrixPanel-I2S-DMA#102]
The library works great on its own!
I have tried to add OTA capability to my code from the Arduino BasicOTA example.
The OTA update is super slow unless I remove matrix.begin(...
Has anyone got this working with OTA?

Thanks

**Comment by mrfaptastic** [comment]
Nope. No issues with OTA here, plenty of people have it working fine.  Probably an issue with that library.

I just use: https://github.com/espressif/arduino-esp32/tree/master/libraries/HTTPUpdate

**Comment by UCSIG** [comment]
I also had lots of problems with OTA updates being really unreliable, slow and failing almost all the time while using the dma matrix library. 
I fixed it by disabling the dma output stream once the update starts by calling `dma_display->stopDMAoutput();` in the `ArduinoOTA.onStart()` callback of the OTA library. Apparently the dma stream and its interrupts were messing with the TCP connection reliability (I saw lots of TCP retransmissions in Wireshark during the update). To reset the ESP32 to a known state after the update, call `ESP.restart();` in the `ArduinoOTA.onError()` and `ArduinoOTA.onend()` callback functions.
